src/h2load.cc

```diff
@@ -1199,6 +1319,12 @@ Worker::~Worker() {
   ev_loop_destroy(loop);
 }
 
+void Worker::stop_all_clients() {
+  for(auto client: clients) {
+    client->~Client();
```

Comment:
This is counter to RAII philosophy, which desirable to have if possible. Ideally, when the worker destructs, all its owned resources, its clients are destructed. How is ownership modeled? Unique/shared ptr?